### PR TITLE
Apache Guide

### PR DESCRIPTION
**Pull request by herpaderpaldent**
* Added Apache Guide for Ubuntu 16.x
* Added Placeholder for Docker installation

### CODE REVIEW

File: docs/installation/webservers/apache.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
+![SeAT](http://i.imgur.com/aPPOxSK.png)
+
+# webserver - apache
+
+The SeAT web interface requires a web server to serve the HTML goodies it has. Apache is an alternative of nginx. You don't **have** to use it, so if you prefer something else, feel free.
```
Comment (leonjza) [inline docs/installation/webservers/apache.md]: > Apache is an alternative of nginx.

Apache is an alternative to nginx.

File: docs/installation/webservers/apache.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+
+!!! info "Docker-Setup"
+
+    This part is still under construction.
+    
+A minimal Apache config might look like this
+
+````
+<VirtualHost *:80>
+    ServerName discourse.example.com
```
Comment (leonjza) [inline docs/installation/webservers/apache.md]: seat.example.com

File: docs/installation/webservers/apache.md [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+
+## host installations
+In order to get the SeAT fronted running, we need to configure Apache to serve our SeAT installs `public/` folder. This is the only folder that should be internet facing. That small `index.php` is the gateway into the application.
+The Apache configuration itself will depend on how your server is set up. Generally, virtual hosting is the way to go, and this is what this guide will be showing here.
+
+
+### Unbuntu Installation
+
+
+Let's install Apache & PHP7.2 mod:
```
Comment (leonjza) [inline docs/installation/webservers/apache.md]: If you are using PHP 7.1, just update the version number to reflect this for libapache2-mod-php7.1

File: docs/installation/webservers/apache.md [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+
+```
+sudo apt-get install apache2 libapache2-mod-php7.2
```
Comment (leonjza) [inline docs/installation/webservers/apache.md]: We always assume you are `root`. Remove `sudo` references please.

File: docs/installation/webservers/apache.md [hunk 5]
```diff
@@ -0,0 +1,197 @@
...
+
+| variable | initial value | new value |
+| ------------ | ------------- | ------------ |
+| ServerAdmin | webmaster@localhost  | email@yourdomain.com |
+| DocumentRoot | /var/www/html  | /var/www/seat |
+| ServerName | -  | yourdomain.com |
+| ErrorLog | ${APACHE_LOG_DIR}/error.log  | ${APACHE_LOG_DIR}/seat-error.log |
+| CustomLog | ${APACHE_LOG_DIR}/access.log combined  | ${APACHE_LOG_DIR}/seat-access.log combined |
+
+Since SeAT is running under a different user other than Apache's `www-data` you might run into some permission issue. Hence using `mpm_itk` module can let Apache run as seat user to avoid permission problem. You don't **have** to use it.
```
Comment (leonjza) [inline docs/installation/webservers/apache.md]: Please remove this. I prefer us using the normal operating system provided users such as `www-data`.

File: docs/installation/webservers/apache.md [hunk 6]
```diff
@@ -0,0 +1,197 @@
...
+
+Finally, enable the site &amp; reload services:
+```
+sudo a2ensite seat.conf
+sudo service apache2 reload 
+```
+
+That's it! SeAT should now be available at http://your-domain-or-ip/
+
+### CentOS Installation
```
Comment (leonjza) [inline docs/installation/webservers/apache.md]: Is this the correct heading? Seems like you are trying to describe a non-vhost setup here.

File: docs/installation/webservers/apache.md (hunk 5 above)
Comment (herpaderpaldent) [reply docs/installation/webservers/apache.md]: i blindly copied this from a privous guide .... shall i completly delete Line 67?

File: docs/installation/webservers/apache.md (hunk 6 above)
Comment (herpaderpaldent) [reply docs/installation/webservers/apache.md]: i copied this guide from a former CentOS manual installation guide. I did not edit anything. The symlink i thought is smth CentOS specific

**Pull request merged**